windows/ReactTestApp/MainPage.h

```diff
@@ -12,6 +12,8 @@ namespace winrt::ReactTestApp::implementation
                               Windows::UI::Xaml::RoutedEventArgs);
         void LoadFromDevServer(Windows::Foundation::IInspectable const &,
                                Windows::UI::Xaml::RoutedEventArgs);
+        winrt::Windows::Foundation::IAsyncAction
```

Comment:
Nit: We're already in `winrt` namespace:
```suggestion
        Windows::Foundation::IAsyncAction
```